src/host/VtIo.cpp

```diff
@@ -155,22 +159,57 @@ VtIo::VtIo() :
             switch (_IoMode)
             {
             case VtIoMode::XTERM_256:
-                _pVtRenderEngine = std::make_unique<Xterm256Engine>(std::move(_hOutput),
-                                                                    initialViewport);
+            {
+                auto xterm256Engine = std::make_unique<Xterm256Engine>(std::move(_hOutput),
+                                                                       initialViewport);
+                if (_passthroughMode)

```

Comment:
Before merge, I am going to wrap this in another layer of `if constexpr` to make sure it is compiled out and all referenced functions are pruned in Windows.